### PR TITLE
[ZH] Update Rules folder and clean up formatting in other languages

### PR DESCRIPTION
**Pull request by huoyaoyuan**
Update song content rules and general rules. Create visual content considerations.

### CODE REVIEW

File: wiki/Rules/Visual_Content_Considerations/zh.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+一个合理应用的例子是来自 Pendulum 的*沉浸式*专辑封面，其在画面中央描绘了裸体男女将手按在一起的倾斜视角。该图标相对较小、典雅、可辨认，由于其非性相关的本质且与该专辑中的歌曲相关，被视为允许。
+
+### 浪漫、调情或亲密的感情
+
+以叙事上下文描绘角色间的浪漫或正常的亲密行为可被接受，如果其没有违反以上规定。
+
+这也包含玩性而不是过激的「轻浮」行为，且不得违反以上规定。
+
+## 更多
+
```
Comment (MegaApplePi) [inline wiki/Rules/Visual_Content_Considerations/zh.md]: It looks like you are missing the contents of this section (see lines 105 - 107 in the English version).

File: wiki/Rules/zh.md [hunk 2]
```diff
@@ -1,59 +1,56 @@
...
 3. **不要讨论被禁言或限制的玩家。** 他的问题只能由他自己和管理员解决，而不是其他人。
-4. **鼓励公平竞争，不要作弊。** 如果你发现了作弊者，请使用`!report`命令举报，或是在他的个人资料页点击举报。
-5. **语言得体，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
+4. **鼓励公平竞争，不要关注作弊玩家。** 很遗憾，作弊玩家确实存在，给予他们过多关注会鼓励他们继续或吸引他人模仿。请使用`!report`命令或是在个人资料页点击来举报他们。
+5. **语言得体，适合全年龄，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
 6. **注意你与他人分享的内容，尤其是可能导致不适的内容。** 分享之前三思而后行。
-7. **确保链接符合法律规定。** 盗版链接，以及“[warez](http://baike.baidu.com/item/WareZ)”相关的内容都不能出现在聊天频道。
-8. **请勿发布第三方聊天工具的链接，包括 Discord，Skype，QQ。** 如果你需要邀请某人加入你的聊天服务器，请在私聊中发。
-9. **在指定了语言的聊天频道，必须使用该语言。** 在没有指定语言的聊天频道，一律使用英文。但是`#lobby`和`#help`允许使用任何语言。*频道管理员在进行工作时可以忽略这条规定。*
+7. **保持链接中的网站相关且合法。** 盗版视频（不包括具有官方授权的视频网站），盗版软件，以及“[warez](http://baike.baidu.com/item/WareZ)”相关的内容都不能出现在聊天频道。可以给游戏直播打广告，但只能在 `#videogames` 频道。
```
Comment (MegaApplePi) [inline wiki/Rules/zh.md]:
```suggestion
7. **保持链接中的网站相关且合法。** 盗版视频（不包括具有官方授权的视频网站），盗版软件，以及“warez”相关的内容都不能出现在聊天频道。可以给游戏直播打广告，但只能在 `#videogames` 频道。
```
remove link to warez

File: wiki/Rules/zh.md [hunk 3]
```diff
@@ -1,59 +1,56 @@
...
+8. **请勿发布第三方聊天工具的链接，包括 Discord，Skype，QQ。** osu! 聊天频道是关于 osu! 本身的，而不是你的聊天服务器。如果你需要邀请某人，请在私聊中发。
```
Comment (MegaApplePi) [inline wiki/Rules/zh.md]: Just noting here, QQ is listed here when it is not in the English version. Should be fine to leave it as it is.

File: wiki/Rules/zh.md [hunk 4]
```diff
@@ -1,59 +1,56 @@
...
+9. **在指定了语言的聊天频道，必须使用该语言（比如在 `#english` 说英文，`#chinese` 说中文）。** 在没有指定语言的聊天频道，一律使用英文。但是 `#lobby` 和 `#help` 允许使用任何语言。*[频道管理员](/wiki/People/Global_Moderation_Team)在进行工作时可以忽略这条规定。*
```
Comment (MegaApplePi) [inline wiki/Rules/zh.md]: noted that this includes example to use \#chinese for Chinese, but should be fine to leave it as it is

File: wiki/Rules/zh.md [hunk 5]
```diff
@@ -1,59 +1,56 @@
...
+这些规定适用于通过游戏内的[谱面提交系统](/wiki/Glossary#bss)提交的谱面。当 osu! 客户端在[编辑器](/wiki/editor)中提交谱面的适合会创建相应的讨论贴。
 
-1. **确保你拥有谱面内所有内容的使用权。** 包括歌曲、视频、击打声、图片以及其它不是你原创的内容。如果你不确定，我们为你提供了一个库，你可以在[杰出艺术家](https://osu.ppy.sh/beatmaps/artists/)列表中找到这个库，它们可以在 osu! 中免费使用。
+1. **确保你拥有谱面内所有内容的使用权。** 包括歌曲、视频、击打声、图片以及其它不是你原创的内容。如果你不确定，我们为你提供了一个不断增长的[精选艺术家](https://osu.ppy.sh/beatmaps/artists/)曲目库，它们全都被授权在 osu! 中使用。
 2. **谱面讨论帖内不要脱离主题。** 这些帖子是专门用来讨论谱面内的问题的。如果你有其它不相关的问题，请在合适的板块中提出。
-3. **不要抄袭他人的成果。** 你不应当在未被授权的情况下上传他人的成果。
+3. **不要抄袭他人的成果。** 你不应当在未被授权的情况下上传他人的成果，包括但不限于皮肤和客串谱面。
+4. **遵守[歌曲内容规定](Song_Content_Rules)和[视觉内容注意事项](Visual_Content_Considerations)。**
 
 ## 当我违反规定时会发生什么？
```
Comment (MegaApplePi) [inline wiki/Rules/zh.md]: missing something from line 49 in the English version

File: wiki/Rules/Song_Content_Rules/zh.md [hunk 6]
```diff
@@ -1,41 +1,42 @@
...
-任何包含违规音频的谱面包都将从网站上删除。
+任何包含违规音频的谱面都将从网站上删除。
 
 每次删除提交的内容时，负责删除的[质量保障团队](https://osu.ppy.sh/groups/7)或[社区管理团队](https://osu.ppy.sh/groups/4)的成员都会向你提供解释，并说明删除的理由。
 
 如果你愿意的话，你可以通过 PM 与他们争论，或者征求其他人的意见来支持你对音频适用性的观点。如果你在这方面仍然找不到共同点，你的要求可能会升级为 NAT 的共识投票，由 NAT 集体决定你的音频是否可以接受。
 
-由于这项裁决的不确定性，所有用户在受到处罚前都有撤回谱面的余地。
+由于这项裁决的不确定性，除了严重的不适上传，所有用户在受到处罚前都有**一次撤回的余地**。
 那些不断上传不合适的曲目的用户将面临 42 小时的处罚，同时删除他们提交的谱面。
```
[the remark this answers is not in the document]
Comment (huoyaoyuan) [reply wiki/Rules/Song_Content_Rules/zh.md]: Yes. The old version didn't merge them.

File: wiki/Rules/zh.md (hunk 5 above)
Comment (huoyaoyuan) [reply wiki/Rules/zh.md]: It's somehow natural to combine line 49 and 51 into one sentence in Chinese. But splitting to achieve line to line matching is also OK.

File: wiki/Rules/zh.md [hunk 7]
```diff
@@ -1,59 +1,58 @@
...
 ## 游戏聊天频道规定
 
 1. **尊重聊天频道内的所有玩家，不要刷屏。** 如果你的聊天占据了半个屏幕，别人就不能愉快的聊天了。
-2. **让频道管理员处理聊天内的问题。** 使用`!report`命令，他们收到后会进行处理。
+2. **让频道管理员处理聊天内的问题。** 使用[`!report`命令](/wiki/Reporting_Bad_Behaviour)，他们收到后会进行处理。
 3. **不要讨论被禁言或限制的玩家。** 他的问题只能由他自己和管理员解决，而不是其他人。
-4. **鼓励公平竞争，不要作弊。** 如果你发现了作弊者，请使用`!report`命令举报，或是在他的个人资料页点击举报。
-5. **语言得体，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
+4. **鼓励公平竞争，不要关注作弊玩家。** 很遗憾，作弊玩家确实存在，给予他们过多关注会鼓励他们继续或吸引他人模仿。请使用`!report`命令或是在个人资料页点击来举报他们。
+5. **语言得体，适合全年龄，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
```
Comment (kj415j45) [inline wiki/Rules/zh.md]:
```suggestion
5. **语言得体，适合全年龄，避免 R18/NSFW 内容。** NSFW 内容并不适合全年龄社区，你可以在私聊中（合情合理地）畅所欲言，但请保持公共频道的清洁。
```

File: wiki/Rules/zh.md [hunk 8]
```diff
@@ -1,59 +1,58 @@
...
-这些规章制度确保了 osu!社区 能有一个有趣而友好的氛围。
+这些规章制度确保了 osu! 社区能对所有人都有一个有趣而友好的氛围。
 
 ## 社区规定
 
-1. **一个玩家同时只能拥有*一个*帐号。** 这个帐号属于你。请勿将你的帐号与他人共享。
-2. **公平竞争。** 不允许使用第三方工具破坏游戏公平性。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。
-3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。
-4. **不要逃避。** 如果你不确定自己是否违反了规定，可能你已经违反了。
+1. **一个玩家全程只能拥有*一个*帐号。** 你注册的*第一个*帐号是你的唯一的 osu! 账号，从注册开始到永远。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建。保持其安全。
```
[the remark this answers is not in the document]
Comment (huoyaoyuan) [reply wiki/Rules/zh.md]: 我感觉这里`一生`有些不妥，看看其他人意见如何

**Comment by cl8n** [comment]
just need a resolution about https://github.com/ppy/osu-wiki/pull/2974#discussion_r367919458 , everything else looks good

File: wiki/Rules/zh.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (cdwcgt) [reply wiki/Rules/zh.md]: 感觉可以去掉，不需要一生或全程
```suggestion
1. **一个玩家只能拥有*一个*帐号。** 你注册的*第一个*帐号是你的唯一的 osu! 账号，从注册开始到永远。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建。保持其安全。
```

[the remark this answers is not in the document]
Comment (cdwcgt) [reply wiki/Rules/zh.md]:
```suggestion
1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建一个新的账号。所以请保持你的账号安全。
```

File: wiki/Rules/zh.md [hunk 9]
```diff
@@ -1,67 +1,66 @@
...
 
-1. **一个玩家同时只能拥有*一个*帐号。** 这个帐号属于你。请勿将你的帐号与他人共享。
-2. **公平竞争。** 不允许使用第三方工具破坏游戏公平性。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。
-3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。
-4. **不要逃避。** 如果你不确定自己是否违反了规定，可能你已经违反了。
-5. **我们是全年龄社区。** 这意味着不能发布 R18/NSFW（Not Safe For Work，不宜在工作期间访问）内容，例如毒品，性。除非它们是为了配合语境，否则绝对不能使用。
-6. **规定中没有指出的，请根据常识做出判断。** 管理员有权利给出他们认为合适的裁决。
+1. **一个玩家全程只能拥有*一个*帐号。** 你注册的*第一个*帐号是你的唯一的 osu! 账号，从注册开始到永远。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建。保持其安全。
+2. **公平竞争。** 不允许使用任何第三方工具获取任何优势。包括宏脚本，辅助瞄准程序，调速工具等一切有利于你在游戏中操作的工具。任何替你做出游戏内本该你自己做出的动作的程序都不行。
+3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲最多八个键，而不是互相恶心。
```
Comment (cdwcgt) [inline wiki/Rules/zh.md]:
```suggestion
3. **待人和善。** 骚扰或敌视他人是不受欢迎的行为，请不要互相伤害。我们在这里点圈圈，敲鼓，接水果，或者同时敲八个键，而不是互相恶心。
```

File: wiki/Rules/zh.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (cdwcgt) [reply wiki/Rules/zh.md]:
```suggestion
1. **一个玩家只能拥有*一个*帐号。从注册开始，** 你注册的*第一个*帐号是你的唯一的 osu! 账号。这个账号代表你。它不是其他任何人————你的父母，你的兄弟姐妹，你的朋友————它就是你。不要与其他任何人共享账号。如果你无法使用该账号，你也不应再创建一个新的账号。所以请保持你的账号安全。
```

File: wiki/Rules/zh.md [hunk 10]
```diff
@@ -1,67 +1,66 @@
...
+4. **不要逃避。** 如果你不确定自己是否违反了规定，恐怕你已经违反了。
+5. **我们是全年龄社区。** 这意味着不能发布 R18/[NSFW](https://zh.wikipedia.org/wiki/NSFW)（Not Safe For Work，不宜在工作期间访问）内容，例如毒品，性。除非它们是为了配合语境，否则绝对不能使用。
```
Comment (cdwcgt) [inline wiki/Rules/zh.md]:
```suggestion
5. **我们是全年龄社区。** 这意味着不能发布 R18/[NSFW](https://baike.baidu.com/item/NSFW/5576169)（Not Safe For Work，不宜在工作期间访问）内容，例如毒品，性。除非它们是为了配合语境，否则绝对不能使用。
```

Comment (huoyaoyuan) [reply wiki/Rules/zh.md]: 目前osu wiki内有多处引用中文wikipedia，应该一起决定和处理

Comment (cdwcgt) [reply wiki/Rules/zh.md]: 了解了

Comment (cl8n) [reply wiki/Rules/zh.md]: I'm going to apply this one, you can search for "wikipedia.org" in zh.md files to easily find the rest (there aren't that many)

**Pull request merged**